### DISCUSSION TITLE
Welcome to Data-Store-Preferences-And-Data-Store-Proto Discussions!

### DISCUSSION THREAD

**Discussion by dheevvvv** [dheevvvv/Data-Store-Preferences-And-Data-Store-Proto#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fridchickenn** [comment]
I'm glad you open the discussion, how are u?

**Reply by dheevvvv** [reply]
Thankyou! I'm good

**Reply by Fridchickenn** [reply]
So in data store it have 2 type, one is preferences and the other is proto right ?

**Reply by dheevvvv** [reply]
yes you right, there is two types in datastore which is preferences and proto
